From d31153b30abd26c3f65e19d126ce4a3542cf14c0 Mon Sep 17 00:00:00 2001
From: Jeremy Huddleston <jeremyhu@apple.com>
Date: Sun, 22 Jan 2012 11:55:29 -0800
Subject: [PATCH 1/2] Include <stdlib.h> for abs()

Tip.c:336:9: error: implicit declaration of function 'abs' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
        ksy += abs(ext->max_ink_extent.y);
               ^
1 error generated.

Signed-off-by: Jeremy Huddleston <jeremyhu@apple.com>
---
 src/Tip.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/src/Tip.c b/src/Tip.c
index 6881670..5f54d14 100644
--- a/src/Tip.c
+++ b/src/Tip.c
@@ -43,6 +43,8 @@
 #include <X11/Xaw3d/TipP.h>
 #include <X11/Xaw3d/XawInit.h>
 
+#include <stdlib.h>
+
 #define	TIP_EVENT_MASK (ButtonPressMask	  |	\
 			ButtonReleaseMask |	\
 			PointerMotionMask |	\
-- 
1.7.7.4

